### PR TITLE
Starting val doesn't appear online

### PR DESCRIPTION
**Pull request by lightest**
Starting val doesn't appear online from Bugs with slider online series.

### CODE REVIEW

**Comment by peircej** [comment]
Task linked: [CU-2au7jh3 Starting val doesn't appear online](https://app.clickup.com/t/2au7jh3)

**Comment by apitiot** [comment]
Awaiting results of comprehensive tests before committing.

File: src/visual/Slider.js [hunk 1]
```diff
@@ -932,9 +932,10 @@ export class Slider extends util.mix(VisualStim).with(ColorMixin, WindowMixin)
 
 		// marker:
 		this._marker = new PIXI.Graphics();
-		if (Number.isFinite(this._startValue))
+		const markerVal = Number.isFinite(this._rating) ? this._rating : (Number.isFinite(this._startValue) ? this._startValue : undefined);
```
Comment (apitiot) [inline src/visual/Slider.js:935]: Please no nested ternary operator :-P
Other than that, I believe we are good to go.

**Review by apitiot — CHANGES_REQUESTED** [review]
See comment regarding nested ternary operators.

**Pull request merged**